modules/utxo-lib/src/bitgo/Musig2.ts

```diff
@@ -0,0 +1,154 @@
+import { PsbtProprietaryKeyValueData, UtxoPsbt } from './UtxoPsbt';
+import { toPlainPublicKey, toXOnlyPublicKey } from './outputScripts';
+import { BIP32Interface } from 'bip32';
+import { musig } from '../noble_ecc';
+
+export const PSBT_IN_MUSIG_PARTICIPANT_PUB_KEYS = 'PSBT_IN_MUSIG_PARTICIPANT_PUB_KEYS';
```

Comment:
I think we should use BITGO as our identifier for all of our proprietary keys, and then use a 1-byte coding for each. We can promote the `Subtype` enum from `ZcashPsbt` to `UtxoPsbt` and add subtypes for each of these musig proprietary fields there.

Reply:
Done